deltachat-ios/DC/Wrapper.swift

```diff
@@ -139,6 +139,16 @@ class DcContext {
         return DcLot(dc_check_qr(contextPointer, qrCode))
     }
 
+    func configureAccountFromQR(qrCode: String) -> Bool {
+        let state = checkQR(qrCode: qrCode).state
+        if state != DC_QR_ACCOUNT {
+            return false
+        }
+        let success = dc_set_config_from_qr(contextPointer, qrCode)
+        configure()
+        return success == 1
```

Comment:
i think, this is wrong. configure() always returns immediately and error/success is reported by DC_EVENT_CONFIGURE_PROGRESS, see AccountSetupController to get an idea, maybe copy/paste the code from there ;)

Reply:
this has been fixed.